app/api/transportations_api.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+class TransportationsApi < Grape::API
+  format :json
+
+  helpers AuthorizationHelper, CarbonFootprintHelper
+
+  namespace 'transportation' do
+    # POST /transportations
+    desc 'Create transportation' do
+      tags %w[transportation]
+      http_codes [
+        { code: 201, message: 'Create transportation' },
+        { code: 400, message: 'Bad request!' }
+      ]
+    end
+    desc 'Headers', {
+      headers: {
+        'Auth-Token' => {
+          description: 'Validates your identity',
+          optional: true
+        }
+      }
+    }
+
+    params do
+      optional :flights, type: Array[JSON] do
+        requires :from, type: String, desc: 'from', documentation: { param_type: 'body' }
+        requires :to, type: String, desc: 'to', documentation: { param_type: 'body' }
+      end
+      optional :public_transports, type: Array[JSON] do
+        requires :transport_type, type: Integer, desc: 'transport_type', documentation: { param_type: 'body' }
+        requires :total_km, type: Float, desc: 'km', documentation: { param_type: 'body' }
+      end
+      optional :cars, type: Array[JSON] do
+        requires :fuel_type, type: Integer, desc: 'fuel_type', documentation: { param_type: 'body' }
```

Comment:
[Same](https://github.com/AmprenTA/backend/pull/3/files#r1023221494), we might add values.